lib/common/socket/evloop.c.h

```diff
@@ -467,15 +467,16 @@ h2o_evloop_t *create_evloop(size_t sz)
     loop->_statechanged.tail_ref = &loop->_statechanged.head;
     update_now(loop);
     /* 3 levels * 32-slots => 1 second goes into 2nd, becomes O(N) above approx. 31 seconds */
-    loop->_timeouts = h2o_timerwheel_create(3, loop->_now);
+    loop->_timeouts = h2o_timerwheel_create(3, loop->_now_ms);
 
     return loop;
 }
 
 void update_now(h2o_evloop_t *loop)
 {
     gettimeofday(&loop->_tv_at, NULL);
-    loop->_now = (uint64_t)loop->_tv_at.tv_sec * 1000 + loop->_tv_at.tv_usec / 1000;
+    loop->_now_ns = (uint64_t)(loop->_tv_at.tv_sec * 1000000 + loop->_tv_at.tv_usec) * 1000;
```

Comment:
I think we should cast `loop->_tv._at.tv_sec` to uint64_t before multiplying it by 1,000,000. Otherwise, the number would overflow if `tv_sec` is a 32-bit integer.